### PR TITLE
add curve file expired process

### PR DESCRIPTION
**Pull request by TristonK**
### What problem does this PR solve?

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by ilixiaocui** [comment]
recheck

**Comment by TristonK** [comment]
recheck

File: src/chunkserver/clone_copyer.cpp [hunk 1]
```diff
@@ -173,9 +217,20 @@ void OriginCopyer::DownloadFromCurve(const string& fileName,
     int fd = 0;
     {
         std::unique_lock<std::mutex> lock(mtx_);
+        std::unique_lock<std::mutex> lockTime(timeMtx_);
         auto iter = fdMap_.find(fileName);
         if (iter != fdMap_.end()) {
             fd = iter->second;
+            auto fdIter = std::find_if(
```
Comment (xu-chaojie) [inline src/chunkserver/clone_copyer.cpp:224]: since the curveOpenTime_ list is orderd, maybe we can use binary search to accelerate.

**Comment by YunhuiChen** [comment]
recheck

**Review by wu-hanqing — APPROVED** [review]

**Pull request merged**